<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tof_powder_plot</class>
 <widget class="QWidget" name="tof_powder_plot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1965</width>
    <height>929</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
   <item>
    <layout class="QVBoxLayout" name="options_layout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QToolButton" name="tB_up">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>previous dataset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tB_down">
         <property name="text">
          <string>next dataset</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="lV_datalist">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="plot_layout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="plot_head_layout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QToolButton" name="tB_grid">
         <property name="toolTip">
          <string>Toggle grid lines</string>
         </property>
         <property name="text">
          <string>grid</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tB_crystal_axes">
         <property name="toolTip">
          <string>Grid on unit cell principal axes</string>
         </property>
         <property name="text">
          <string>crystal_axes</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tB_projections">
         <property name="toolTip">
          <string>Toggle projections</string>
         </property>
         <property name="text">
          <string>projections</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tB_linestyle">
         <property name="toolTip">
          <string>Toggle edgecolor of triangles/quadliterals or change pointsize for scatter plot</string>
         </property>
         <property name="text">
          <string>linestyle</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tB_save_data">
         <property name="toolTip">
          <string>export data, to directory specified in path</string>
         </property>
         <property name="text">
          <string>save_data</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="plot_footer_layout">
       <item>
        <widget class="QLabel" name="l_xrange">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lE_xrange">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l_yrange">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lE_yrange">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l_zrange">
         <property name="text">
          <string>Z</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lE_zrange">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tB_log">
         <property name="text">
          <string>log</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combB_colormap">
         <property name="iconSize">
          <size>
           <width>100</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tB_invert_cb">
         <property name="text">
          <string>invert</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l_fontsize">
         <property name="text">
          <string>FontSize:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sB_fontsize">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="value">
          <number>18</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
